.github/workflows/build.yml

```
@@ -408,3 +408,8 @@ jobs:
           retention-days: 7
           if-no-files-found: ignore
           include-hidden-files: true
+
+      - name: Compare SimpleCov results with Undercover
+        run: |
+          git fetch origin master:master
```

I noticed that Git is fetching all the tags. We can skip that:
```suggestion
          git fetch --no-tags origin master:master
```